corehq/apps/cloudcare/templates/formplayer/case_list.html

```diff
@@ -244,19 +244,25 @@ <h1  aria-label="<%- title %>" tabindex="0" class="page-title"><%- title %></h1>
       grid-area: <%- model.gridStyle %>;
       font-size: <%- model.fontStyle %>;
       text-align: <%- model.horizontalAlign %>;
-      <% if (!model.borderInTile) { %>
+      <% if (!model.borderInTile && !model.shadingInTile) { %>
         justify-self: <%- model.horizontalAlign %>;
         align-self: <%- model.verticalAlign %>;
```

Comment:
Are these properties just not used/usable when the tile has a border or shading applied?

Reply:
Yes, those properties are not used when the tile has either shading, border or both. If shading or border are set then `justify-self` is set to `stretch` and the padding, margin etc are set. (Personally I find the ejs syntax almost unreadable)